lib/modulr/api/accounts_service.rb

```diff
@@ -3,13 +3,19 @@
 module Modulr
   module API
     class AccountsService < Service
-      def create(customer_id:, currency:, product_code:, options: {})
-        data = {
+      def find(id:)
+        response = client.get("/accounts/#{id}")
+        Resources::Accounts::Account.new(response, response.body)
+      end
```

Comment:
Custom naming has been removed, so there is no `info` anymore. The usual CRUD ops have been defined as `find` (read resource), `list` (read collection resource) and `create`  (create resource).